docs/framework/react/guide/code-splitting.md

```diff
@@ -180,6 +180,8 @@ Tada! 🎉
 
 > ⚠️ **Experimental:** This feature is experimental and currently in development. It is not recommended for production use.
 
+> ⚠️ **Important:** This feature is currently does not work with Rspack/Rsbuild. If you are familiar with [Unplugin](https://unplugin.unjs.io) or Rspack, please let us know if you can help us get this working!
```

Comment:
Glad to see that TanStack Router now supports Rspack.

For th unplugin issue mentioned here, it seems to be caused by Rspack not supporting the unplugin resolveId hook. I have reported this to the Rspack team and expect to see support for it in the future.